substra/cli/interface.py

```diff
@@ -148,6 +148,17 @@ def click_option_verbose(f):
     )(f)
 
 
+def click_option_permissions(f):
+    """Add permissions file path to command."""
+    return click.option(
+        '--permissions-path',
+        'permissions',
+        type=click.Path(exists=True, dir_okay=False), callback=load_json_from_path,
```

Comment:
Could you move callback to its own line just to make it clear it's not part of the type?